pandas/tests/frame/test_repr_info.py

```diff
@@ -332,13 +332,25 @@ def test_info_memory_usage(self):
         res = buf.getvalue().splitlines()
         assert re.match(r"memory usage: [^+]+$", res[-1])
 
-        assert (df_with_object_index.memory_usage(
-            index=True, deep=True).sum() > df_with_object_index.memory_usage(
-            index=True).sum())
```

Comment:
split into 2 tests, 1 for PYPY 1 for existing (skip for PYPY on existing, skip if not PYPY on other test)